tests/Services/Search/ParametersHandlerTest.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+namespace Findologic\Tests\Services\Search;
+
+use Ceres\Helper\ExternalSearchOptions;
+use Findologic\Services\Search\ParametersHandler;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Plenty\Plugin\Http\Request;
+
+/**
+ * Class SearchServiceTest
+ * @package Findologic\Tests
+ */
+class ParametersHandlerTest extends TestCase
+{
+    public function providerGetSortingOptions()
```

Comment:
We should use the same naming convention for data provider as we do everywhere: type of returned data + "Provider" e.g. `sortingOptionsProvider`

Reply:
But in other files naming convention is 'provider' + 'name of the test'. Should I rename in other places as well?

Reply:
That one must have slipped through. Please change both occurrences.